src/btllib/include/btllib/graph.hpp

```diff
@@ -0,0 +1,96 @@
+// #ifndef BTLLIB_GRAPH_HPP
```

Comment:
If this isn't ready (everything is commented out), shouldn't you just leave it out of the PR?

Reply:
As btllib is still being worked on that file exists there and is not a specific to this PR. I guess your comment is more relevant to btllib itself.